### PR TITLE
Fix RTL6g1 test.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

File: ablySpec/RealtimeClientChannel.swift [hunk 1]
```diff
@@ -950,7 +950,7 @@ class RealtimeClientChannel: QuickSpec {
                             expect(messageSent.messages![0].clientId).to(beNil())
 
                             let messageReceived = transport.protocolMessagesReceived.filter({ $0.action == .Message })[0]
-                            expect(messageReceived.messages![0].clientId).to(beNil())
+                            expect(messageReceived.messages![0].clientId).to(equal(options.clientId))
```
Comment (ricardopereira) [inline ablySpec/RealtimeClientChannel.swift]: I don't agree with this change. It should be nil...

[RTL6g1b](http://docs.ably.io/client-lib-development-guide/features/#RTL6g1a)
> the clientId value is null for the Message when received

Comment (tcard) [inline ablySpec/RealtimeClientChannel.swift]: Yes, this changed recently (https://github.com/ably/docs/pull/91), it should've commented it in the commit.

Comment (mattheworiordan) [inline ablySpec/RealtimeClientChannel.swift]: Yup and now deployed, see http://docs.ably.io/client-lib-development-guide/features/#RTL6g1b

**Comment by mattheworiordan** [comment]
LGTM

**Comment by tcard** [comment]
@ricardopereira PTAL

**Comment by ricardopereira** [comment]
:+1:

**Pull request merged**